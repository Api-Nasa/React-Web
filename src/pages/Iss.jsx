import React from 'react';
import PantallaPresentacion from '../components/PantallaPresentacion';

function Iss() {
  const data = {
    urlIcono: 'https://i.gifer.com/ONml.gif',
    altIcono: 'Iss',
    link: '/MapsEstacionIss',
    tituloPresentacion:
      ' Discover the real-time location of the international space station',
  };
  return (
    <div>
      <PantallaPresentacion data={data} />
    </div>
  );
}

export default Iss;

