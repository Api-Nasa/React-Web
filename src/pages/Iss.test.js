
/* importaciones necesarias para este test */
import React from 'react';
import {BrowserRouter as Router} from 'react-router-dom';
import { render } from '@testing-library/react';
import Iss from './Iss';

/* probamos instantanea o snapshot */


test('react component snapshot', () => {
  const view = render(
    <Router>
      <Iss/>
    </Router>,
  );
  expect(view).toMatchSnapshot();
});


  