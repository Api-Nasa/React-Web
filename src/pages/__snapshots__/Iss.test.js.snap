// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`react component snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App animate__animated animate__zoomIn"
      >
        <header
          class="App-header"
        >
          <a
            href="/MapsEstacionIss"
          >
            <button
              class="btn btn-primary iss-btn "
              type="button"
            >
              Get started for discover
            </button>
          </a>
          <img
            alt="Asteriode"
            src="https://i.gifer.com/ONml.gif"
          />
          <div
            class="titulo-iss animate__animated animate__zoomIn "
          >
            <img
              alt="gif atom"
              class="App-logo logo-peq "
              src="atom.gif"
            />
            <p
              class="titulo-landing "
            >
              Discover the real-time location of the international space station
               
            </p>
          </div>
        </header>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App animate__animated animate__zoomIn"
    >
      <header
        class="App-header"
      >
        <a
          href="/MapsEstacionIss"
        >
          <button
            class="btn btn-primary iss-btn "
            type="button"
          >
            Get started for discover
          </button>
        </a>
        <img
          alt="Asteriode"
          src="https://i.gifer.com/ONml.gif"
        />
        <div
          class="titulo-iss animate__animated animate__zoomIn "
        >
          <img
            alt="gif atom"
            class="App-logo logo-peq "
            src="atom.gif"
          />
          <p
            class="titulo-landing "
          >
            Discover the real-time location of the international space station
             
          </p>
        </div>
      </header>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
